eclass/coreos-kernel.eclass

```diff
@@ -7,6 +7,16 @@
 # Revision of the source ebuild, e.g. -r1. default is ""
 : ${COREOS_SOURCE_REVISION:=}
 
+# @ECLASS-VARIABLE: COREOS_TARGET_BOARD
```

Comment:
Since the hikey build is targeting an older kernel anyway we can just leave this out for now, if needed later I'd favor a USE flag or even a different ebuild name than magic environment variables.